python/nx-cugraph/nx_cugraph/classes/graph.py

```diff
@@ -58,7 +59,7 @@ class Graph:
     node_values: dict[AttrKey, cp.ndarray[NodeValue]]
     node_masks: dict[AttrKey, cp.ndarray[bool]]
     key_to_id: dict[NodeKey, IndexValue] | None
-    _id_to_key: dict[IndexValue, NodeKey] | None
+    _id_to_key: list[NodeKey] | None
```

Comment:
Is this a breaking change? If so, I'll add the `breaking` label

Reply:
Technically, yes, practically, no. Nearly all internal uses of it was as `G.id_to_key[index]`, which works whether a dict or a list. It is breaking if one did other things such as `G.id_to_key.items()`.

Reply:
ok I'll leave off the "breaking" label then.